vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -161,6 +161,8 @@ def _get_build_configuration(self, app_type, working_directory):
             build_configuration = BuildConfiguration(app_type, working_directory)
         elif app_type == 'AspNetCore':
             build_configuration = BuildConfiguration(app_type, working_directory)
+        elif app_type == 'PHP':
```

Comment:
Converting 'app_type' parameter value to Upper Case while parsing itself, hence user can type-in case insensitive values.